service-node/backend/src/test/scala/org/enterprisedlt/fabric/service/node/UtilTest.scala

```diff
@@ -85,4 +87,31 @@ class UtilTest extends FunSuite {
     }
 
 
+    test("fillPlaceholdersEnvironmentVariables should work fine") {
```

Comment:
- please change caption to "what exactly test does" i.e. "fillPlaceholdersProperties should replace single placeholder"
- please name values in the body accordingly to meaning

Reply:
done.